GHzDACs/Cleanup/dac.py

```diff
@@ -765,21 +782,33 @@ def setPolarity(self, chan, invert):
         return self.testMode(self._setPolarity, chan, invert)
 
     def setLVDS(self, cmd, sd, optimizeSD):
-        """
-        INPUTS
-        cmd - int: 2 for DAC A, 3 for DAC B
-        sd - :
-        optimizeSD - bool:
-        
-        Returns (success, MSD, MHD, t, (range(16), MSDbits, MHDbits), checkHex)
-        success - bool: 
-        MSD - int:
-        MHD - int:
-        t - :
-        range(16)
-        MSDbits - :
-        MHDbits - :
-        checkHes - :
+        """ Calibrate DAC LVDS
+        Align DAC clocks for LVDS, varying SD (sample delay).
+        :param int cmd: 2 for DAC A, 3 for DAC B
+        :param int/None sd: If int, SD value to be set. If None, SD
+                            set to value in registry. Ignored if
+                            optimizeSD=True.
+        :param bool optimizeSD: Whether to follow procedure from data
+                                sheet to determine SD
+        :return: success - If LVDS bringup successfull. MSD and MHD
+                           should only flip once with flip locations
+                           within one bit of each other. If varies,
+                           could mean clock noise.
+                 MSD - Measured sample delay. If optimizeSD, where MSD
+                       flips when MHD=SD=0. Else -1.
+                 MHD - Measured hold delay. If optimizeSD, where MHD
+                       flips when MSD=SD=0. Else -1.
+                 t - SD value set
+                 tuple - Transpose of [SD, MSD(SD), MHD(SD) for SD in range(16)]
+                 checkHex - In binary, '0bABC', where
+                            A=1: An LVDS input above the IEEE input
+                                 voltage specification limit
+                            B=1: An LVDS input below the IEEE input
+                                 voltage specification limit
+                            C=1: Sampling in correct data cycle
```

Comment:
Uh, er, well...

To quote the data sheet:
```
0, phase measurement—sampling in the previous or following DATA cycle.
1, phase measurement—sampling in the correct DATA cycle.
```
Perhaps referring to one of the DATA clocks? But which one (maybe DATACLK_IN, but not sure).